content-repo/gendocs.py

```diff
@@ -998,9 +998,10 @@ def main():
     if os.getenv('UPDATE_PACK_DOCS') or os.getenv('CI'):
         # to avoid cases that in local dev someone might checkin the modifed pack-docs.md we do this only if explicityl asked for or in CI env
         insert_approved_tags_and_usecases()
-    if os.getenv('UPDATE_TOP_CONTRIBS') or os.getenv('CI'):
-        print("Writing json links into contentItemsDocsLinks.json")
-        update_contributors_file(create_service_account_file().name, json.dumps(DOCS_LINKS_JSON, indent=4))
+    # To avoid updating on preview website (non-master branches) since we generate only 20 reference pages (and links) for each category.
+    if os.getenv('CURRENT_BRANCH') == 'master':
```

Comment:
CURRENT_BRANCH env is set in [create_docs](https://github.com/demisto/content-docs/blob/master/content-repo/create-docs.sh#L32) script which [calls](https://github.com/demisto/content-docs/blob/master/content-repo/create-docs.sh#L158) the gen_docs.py (current) script.